prow/tide/tide.go

```diff
@@ -1844,14 +1844,65 @@ type PullRequest struct {
 	// additional API token. (see the 'headContexts' func for details)
 	// We can't raise this too much or we could hit the limit of 50,000 nodes
 	// per query: https://developer.github.com/v4/guides/resource-limitations/#node-limit
-	Commits   Commits `graphql:"commits(last: 4)"`
+	Commits   Commits `graphql:"commits(last: 100)"`
```

Comment:
Replace the custom list pr commits method implements by increase the limit from 4 to 100.

@Mini256 @hi-rustin PTAL